src/core/selection/selection.model.js

```diff
@@ -3,6 +3,7 @@ import Resource from 'core/resource/resource';
 export default class SelectionModel {
 	constructor() {
 		this.mode = 'row';
```

Comment:
can you write what modes we are supporting in comments here like this.mode = 'row'; //row | cell| column ?